### PR TITLE
Update to 1.13 and fix api endpoint for determining the country for an IP address 

### PR DESCRIPTION
**Pull request by frost-byte**
Updated a few dependencies, bumped the version and fixed Country lookup by IP

### CODE REVIEW

File: src/main/java/me/leoko/advancedban/bukkit/BukkitMethods.java [hunk 1]
```diff
@@ -75,12 +76,15 @@ public String getFromUrlJson(String url, String key) {
             JSONParser jp = new JSONParser();
             JSONObject json = (JSONObject) jp.parse(new InputStreamReader(request.getInputStream()));
 
-            String[] keys = key.split("\\|");
```
Comment (DevLeoko) [inline src/main/java/me/leoko/advancedban/bukkit/BukkitMethods.java]: This code enabled the user to define a key which points to a child JsonObject.
Like result|playerdata|uuid
would point to result: { playerdata: { uuid: VALUE}}
Is there any reason for you to remove it? ^^

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>me.leoko.advancedban</groupId>
     <artifactId>AdvancedBan</artifactId>
-    <version>2.1.6-${build}</version>
```
Comment (DevLeoko) [inline pom.xml]: Keep the version an 2.1.6 as it's not released yet

**Comment by frost-byte** [comment]
I fixed those issues.  Sorry about the mix up with version and the mix up with getUrlFromJson.

**Comment by DevLeoko** [comment]
You added two repos and one dependency but you don't seem to actually ever use them.

**Comment by frost-byte** [comment]
I was unable to find the newer version of RedisBungee at the repository you were using.
The repo at [ci.lucko.me](https://ci.lucko.me/plugin/repository/everything/com/imaginarycode/minecraft/RedisBungee/) has a build of v0.4

If you'd like, I'll just set it back to 0.3.6-SNAPSHOT and remove the above repo.

I'll remove the repo/dependency for net.sourceforge.nekohtml.

**Comment by frost-byte** [comment]
You're welcome. Thank you for merging it.

**Pull request merged**